pages/linux/zypper.md

```diff
@@ -0,0 +1,23 @@
+# zypper
+
+> SUSE & openSUSE package management utility
+
+- Synchronize list of packages and versions available.
+
+`zypper refresh`
+
+- install a new package
```

Comment:
Please capitalize the first letter of each sentence (e.g. "Install" here, and "Remove" on next item).